pkg/crc/machine/machine.go

```diff
@@ -204,6 +205,21 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		return *result, err
 	}
 
+	// kubeconfig file is also needed inside the VM to perform pull secret replace operation.
```

Comment:
@cfergeau no, I wished we have one :(

Reply:
@cfergeau i am bit confuse should we make it part of `snc` script?

Reply:
I would say no. Actually, now I'm wondering, since we are starting the kubelet in the VM, can't we run the `oc replace` from the host?

Reply:
@cfergeau the way we are right now doing it in the vm using below command. which make sure as soon as it get the secret resource just replace it and the soon stop the kublet service. 

```
$ timeout 80 bash -c 'sudo systemctl start kubelet && \
until oc --config /tmp/kubeconfig replace -f /tmp/pull-secret.yaml 2>/dev/null 1>&2; \
do echo "Waiting for recovery apiserver to come up."; sleep 1; done'
```

we can do the refactor and start using from the host but I am bit spectacle about it since on the host there are still some of unknown (like firewall or dns working etc.) once we do it from guest it is much reliable.